### ISSUE TITLE
hostname independent solution

### ISSUE THREAD

**Issue by Silv-1** [jbreuer/1-1-multilingual-example#11]
Wouldn't it be better when you didn't have to set any hostnames and it just takes the languages specified in settings?

My problem is, that we can't call the website with other domains than explicitly configured in hostnames, but we want to manage our domains on our server and don't have to always also set them in umbraco hostnames.

**Comment by jbreuer** [comment]
Yes it's also possible to change the code to use the language specific settings. For the example using the hostnames was fine, but there might be better solutions.
